### ISSUE TITLE
question: Does it work with migrations at all?

### ISSUE THREAD

**Issue by capaj** [czeidler/knex-pglite#2]
getting when I try to run migrate in a sample knex project 

```

> thefork@1.0.0 migration
> knex migrate:latest

Requiring external module /home/capaj/Downloads/code (3)/code/restaurant-service/node_modules/interpret/mjs-stub
knex: Unknown configuration option 'client' value [object Object]. Note that it is case-sensitive, check documentation for supported values.
Error: knex: Unknown configuration option 'client' value [object Object]. Note that it is case-sensitive, check documentation for supported values.
    at resolveConfig (/home/capaj/Downloads/code (3)/code/restaurant-service/node_modules/knex/lib/knex-builder/internal/config-resolver.js:32:13)
    at knex (/home/capaj/Downloads/code (3)/code/restaurant-service/node_modules/knex/lib/knex-builder/Knex.js:14:39)
    at initKnex (/home/capaj/Downloads/code (3)/code/restaurant-service/node_modules/knex/bin/cli.js:84:10)
    at async Command.<anonymous> (/home/capaj/Downloads/code (3)/code/restaurant-service/node_modules/knex/bin/cli.js:249:26)
(base)  ✘ capaj@capaj-ryzen-7-5900x  ~/Downloads/code (3)/code/restaurant-service  

```

**Comment by czeidler** [comment]
In principle it "should" work. Which sample knex project do you mean? what are the steps to reproduce it?

**Comment by ghchaosfae** [comment]
You can make it work correctly for now by passing `ClientPgLite.default` as the client

Example:
```js
import ClientPgLite from "knex-pglite";

export default {
	client: ClientPgLite.default,
	connection: {
		filename: "./data",
	},
	migrations: {
		directory: "./migrations",
		loadExtensions: [".mjs"],
	},
};
```

https://github.com/czeidler/knex-pglite/issues/3

**Comment by czeidler** [comment]
This should be fixed now (also see #3).
